### PR TITLE
Update KDE sdk to latest used by upstream

### PR DESCRIPTION
**Pull request by pm4rcin**
I have updated the SDK to the latest that AppImage uses. Should be tested if everything works correctly.

### CODE REVIEW

**Comment by pm4rcin** [comment]
From quick testing things are dowloaded and shared. I've had 2 crashes on sway but not sure if it's Qt or something else. If someone could test it would be great.

**Comment by ReillyBrogan** [comment]
FWIW updating to qt6 will cause a worse experience for Plasma users as the built-in theme Breeze does not support qt6 and will not until the release of Plasma 6. I'd recommend sticking with qt5-based runtime builds until Plasma 6 is out.

File: org.qbittorrent.qBittorrent.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ app-id: org.qbittorrent.qBittorrent
 default-branch: stable
 runtime: org.kde.Platform
 sdk: org.kde.Sdk
-runtime-version: 5.15-22.08
+runtime-version: 6.6
```
Comment (Erick555) [inline org.qbittorrent.qBittorrent.yaml:5]: use `5.15-23.08` for now and remove `DQT6=ON` below

**Comment by Chocobo1** [comment]
@pm4rcin and every participant here
Thank you!

**Pull request merged**